packages/ui/src/__tests__/atoms/Dropdown.test.tsx

```diff
@@ -18,34 +32,32 @@ describe('Dropdown', () => {
     expect(dropdown).toBeInTheDocument()
     expect(dropdown).toHaveStyle(`cursor: pointer;`)
     expect(dropdownHeader).toHaveTextContent(/selecciona/i)
+    await userEvent.click(dropdownHeader)
     expect(screen.getByTitle('Ampliar')).toBeInTheDocument()
     expect(screen.queryByText('Test children content')).not.toBeInTheDocument()
     expect(screen.queryByTitle('Cerrar')).not.toBeInTheDocument()
   })
 
   it('renders dropdown children when user clicks on it', async () => {
     render(
-      <Dropdown>
-        <p>Test children content</p>
-      </Dropdown>
+      <Dropdown options={mockOptions} />
     )
-
     const dropdownHeader = screen.getByTestId('dropdown-header')
 
     expect(screen.queryByText('Test children content')).not.toBeInTheDocument()
-
+    
+    await userEvent.click(dropdownHeader)
     expect(screen.getByTitle('Ampliar')).toBeInTheDocument()
+    
+    expect(screen.queryByText('Test children content')).toBeVisible() 
```

Comment:
'Test children content' was text in children, you have to update it with some text in options